### PR TITLE
Update wireguard and kernels (4.13.11/4.9.60/4.4.96)

### PR DESCRIPTION
**Pull request by rn**
- Update `linuxkit/alpine` to pick up the lates WireGuard tools
- Update `getty`/`ip`/`sshd` packages to pick up new WireGuard tools
- Update kernels to 4.13.11/4.9.60/4.4.96

Note the latest WireGuard kernel module does not compile for the our debug kernels. This is fixed upstream and should be picked up with the next WireGuard release. In the meantime cherry-pick the  patch fixing the compile error.
wip #2669

penguin

### CODE REVIEW

**Comment by rn** [comment]
x86_64 kernels are already compiled/pushed. aarch64 kernels are being compiled/pushed.

File: examples/bar/mac-addr [hunk 1]
```diff
@@ -0,0 +1 @@
+e2:14:27:53:9f:9f
```
Comment (riyazdf) [inline examples/bar/mac-addr]: what is `examples/bar` for?

File: examples/bar/qemu.pid [hunk 2]
```diff
@@ -0,0 +1 @@
+87422
```
Comment (riyazdf) [inline examples/bar/qemu.pid]: same here about `examples/bar`

File: examples/foo/mac-addr [hunk 3]
```diff
@@ -0,0 +1 @@
+9e:70:33:12:ee:0a
```
Comment (riyazdf) [inline examples/foo/mac-addr]: what is `examples/foo` for?

File: examples/foo/qemu.pid [hunk 4]
```diff
@@ -0,0 +1 @@
+87412
```
Comment (riyazdf) [inline examples/foo/qemu.pid]: same here for `examples/foo`

File: examples/bar/mac-addr (hunk 1 above)
Comment (rn) [reply examples/bar/mac-addr]: crap, some old files got added. Force pushed an update

**Comment by rn** [comment]
rebased to include travis fix

**Pull request merged**